### DISCUSSION TITLE
UI suggestion

### DISCUSSION THREAD

**Discussion by Hacker-web-Vi** [subquery/subql#1022] [Frontier TestNet - Bug Reporting]
![Search](https://user-images.githubusercontent.com/80550154/167955447-a78794a5-63a7-46ed-8412-607c9fc4b1e5.png)

Hi, SubQuery team!
Looks like all bugs have been reported
I would just suggest adding some **search box + sorting indexers option** (by comission rate/uptime/delegation) on https://frontier.subquery.network/staking/indexers page
I've attached the image as an example 

Best regards, DanielMoon#9292

**Comment by mzxyz** [comment]
- Thanks for you suggestion, searching feature is under development, please expect this will be included in the next explorer updates 
- @lilian79511  as @Hacker-web-Vi suggested, have filter of (comission rate/ indexer available  etc)  in the page would be a good idea

**Comment by heyacherry** [comment]
Hello @Hacker-web-Vi 
Thank you for your suggestions！The search and sorting feature has been released.
